reporter/errors.go

```diff
@@ -30,26 +30,29 @@ var ErrInvalidSource = errors.New("parse failed: invalid proto source")
 // about the location in the file that caused the error.
 type ErrorWithPos interface {
```

Comment:
Since we're making a backwards-incompatible change anyway, maybe rename to `ErrorWithSpan`?

Reply:
As long as no one is creating there own ErrorWithPos impls, leaving this name, and the GetPosition() function, should allow upgrades without code changes right? If so, that seems a likely scenario, and valuable.